### PR TITLE
Ira

### PR DESCRIPTION
**Pull request by irinakedem1**
read sentry dsn from env

### CODE REVIEW

File: src/components/LoginPage.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import React, { Component } from 'react';
 import {kc,getUser} from './UserManager';
 import { Container,Message,Button,Dropdown } from 'semantic-ui-react';
 
+//test
+
```
Comment (amnonbb) [inline src/components/LoginPage.js]: Remove it

File: src/components/LoginPage.js [hunk 2]
```diff
@@ -16,7 +18,7 @@ class LoginPage extends Component {
     appLogin = () => {
         getUser((user) => {
             if(user) {
-                this.setState({loading: false});
+                this.setState({loading: false})
```
Comment (amnonbb) [inline src/components/LoginPage.js]: Add ;

File: src/shared/sentry.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        // We recommend adjusting this value in production, or using tracesSampler
+        // for finer control
+        tracesSampleRate: 1.0,
+    });
+}
+
+export const initSentry2 = () => {
+    Sentry.init({
+        //dsn: `https://${SENTRY_KEY}@sentry.kli.one/2`,
+        dsn: "https://0072ad3d8e9345f685c9b2378119c9cb@sentry.kli.one/3",
```
Comment (amnonbb) [inline src/shared/sentry.js]: Don't leave sensitive info in git. Maybe remove all function if it's not used

File: src/shared/sentry.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        //release: version,
+        environment: process.env.NODE_ENV,
+        attachStacktrace: true,
+        maxBreadcrumbs: 100,
+    });
+
+    Sentry.configureScope((scope) => {
```
Comment (amnonbb) [inline src/shared/sentry.js]: We don't use this in our project

**Review by amnonbb — CHANGES_REQUESTED** [review]

File: src/shared/sentry.js (hunk 4 above)
Comment (irinakedem1) [reply src/shared/sentry.js]: done

File: src/shared/sentry.js (hunk 3 above)
Comment (irinakedem1) [reply src/shared/sentry.js]: done

File: src/components/LoginPage.js (hunk 2 above)
Comment (irinakedem1) [reply src/components/LoginPage.js]: done

File: src/components/LoginPage.js (hunk 1 above)
Comment (irinakedem1) [reply src/components/LoginPage.js]: done

**Pull request merged**